docs/alerts.mdx

```diff
@@ -107,6 +107,45 @@ Helpful resources for learning how to use SpEL:
     </TabItem>
 </Tabs>
 
+### LuckPerms
+:::warning
+Requires the [LuckPermsAlerts](https://github.com/ANutley/LuckPermsAlerts) plugin
+:::
+
+
+<Tabs>
+    <TabItem value="luckperms-nodeaddevent" label="NodeAddEvent (Permission)" default>
+        [NodeAddEvent.java](https://github.com/LuckPerms/LuckPerms/blob/master/api/src/main/java/net/luckperms/api/event/node/NodeAddEvent.java)      
+        ```yaml
+        # Example alert to send a message when a permission has been added or removed from a player
+        - Trigger: LPNodeAddEvent
+          Channel: global
+          Conditions:
+          - "#event.getEvent().getNode().getType() == 'PERMISSION'"
+          - "#event.getEvent().isUser()"
+          Embed:
+            Title: "Player's permissions changed"
+            Color: "#00ff00"
+            Description: "`${#event.getEvent().getTarget().getFriendlyName()}` has had the permission `${#event.getEvent().getNode().getPermission()}` set to `${#event.getEvent().getNode().getValue()}`"
```

Comment:
```suggestion
          - Trigger: LPNodeAddEvent
            Channel: global
            Conditions:
            - "#event.getEvent().getNode().getType() == 'PERMISSION'"
            - "#event.getEvent().isUser()"
            Embed:
              Title: "Player's permissions changed"
              Color: "#00ff00"
              Description: "`${#event.getEvent().getTarget().getFriendlyName()}` has had the permission `${#event.getEvent().getNode().getPermission()}` set to `${#event.getEvent().getNode().getValue()}`"
```
Indented up by two spaces to allow end users to paste this directly into the alerts.yml file with no indentation issues